lms/djangoapps/mobile_api/course_info/views.py

```diff
@@ -104,9 +104,12 @@ class CourseAboutDetail(generics.RetrieveAPIView):
 
         * overview: The HTML for the course About page.
     """
+    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    permission_classes = (permissions.IsAuthenticated,)
 
-    @mobile_course_access(verify_enrolled=False)
-    def get(self, request, course, *args, **kwargs):
+    def get(self, request, *args, **kwargs):
+        course_id = CourseKey.from_string(kwargs['course_id'])
+        course = modulestore().get_course(course_id)
```

Comment:
The changes in this file are not intended, correct?  This reverts changes made in this PR: https://github.com/edx/edx-platform/pull/6398.  Please remove.